docs/build-haiku.md

```diff
@@ -0,0 +1,44 @@
+Haiku builds can be created using Clang or GCC using Meson buildsystem.
+
+## Build on Haiku
+
+Install dependencies:
+
+``` shell
+pkgman install -y meson ccache libpng libsdl2_devel sdl2_net_devel \
+                  libogg_devel opus_devel opusfile_devel gcc_syslibs_devel \
+                  opus_tools fluidsynth2_devel llvm_clang
+```
+
+Clone and enter the repository's directory:
+
+``` shell
+git clone https://github.com/dosbox-staging/dosbox-staging.git
+cd dosbox-staging
+```
+
+Build with OpenGL disabled (Haiku supports OpenGL only via software fallback):
+
+``` shell
+meson setup -Duse_opengl=false build
+ninja -C build
+```
+
+Detailed instructions and build options are documented in [BUILD.md](/BUILD.md).
+
+## Run
+
+Edit your configuration file by running: `dosbox -editconf` and make the
+following suggested changes (leave all other settings as-is):
+
+``` ini
+[sdl]
+output = texture
```

Comment:
`output = texture  # or texturenb, texturepp`

Reply:
But we don't want to promote usage of `texturenb` :) This setting works only when .conf file is very specifically tuned, not text mode is ever used, scaling is by integer only, and game will never switch resolution on its own… If any of those do not occur - `nb` is going to introduce bad visual artifacts. And it's not faster than normal `texture`.

We don't want to promote `texturepp` either - it's very subjective and we want to replace it by a new setting.

The only reason for not removing this `ini` example completely (`texture` is default when compiling without OpenGL) is: I never tested it on Haiku and don't know if they have their own HW-backend better than `software` or not.